ansible/library/load_extra_dpu_config.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import paramiko
+import os
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.smartswitch_utils import smartswitch_hwsku_config
+
+DPU_HOST_IP_BASE = "169.254.200.{}"
+SRC_DPU_CONFIG_FILE = "/tmp/dpu_extra.json"
+DST_DPU_CONFIG_FILE = "/tmp/dpu_extra.json"
```

Comment:
SRC_DPU_CONFIG_FILE and DST_DPU_CONFIG_FILE have same value?

Reply:
SRC_DPU_CONFIG_FILE  on NPU, DST_DPU_CONFIG_FILE  on DPU